contracts/Example3.sol

```diff
@@ -1,14 +1,13 @@
 pragma solidity ^0.8.4;
 
 contract Example3 {
-    uint256 marginPercentage = 30;
     function getOwnerMargin(uint256 amount)
-        public
-        view
+        external
+        pure
         returns (uint256 amountForSender, uint256 amountForOwner)
     {
-        amountForSender = (amount * (100 - marginPercentage)) / 100;
-        amountForOwner = (amount * marginPercentage) / 100;
+        amountForSender = (amount * 70) / 100;
```

Comment:
Better to have `marginPercentage` stored in a variable since it is used in two places (directly or indirectly). To avoid accessing storage twice you can move `uint256 marginPercentage = 30;` to inside the function